index.html

```diff
@@ -2,18 +2,86 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+    <meta charset="utf-8">
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <title>Great Idea!</title>
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--[if lt IE 9]>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
 </head>
-
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+    <header>
+        <nav>
+            <a href="#">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </nav>
+        <a class="logo-anchor" href="#">
```

Comment:
You could put index.html in the href for this anchor tag instead of having it as a dead link 😄